include/mesh.tcc

```diff
@@ -317,6 +317,76 @@ void mpm::Mesh<Tdim>::compute_cell_neighbours() {
       tbb::simple_partitioner());
 }
 
+//! Find particle neighbours for all particle
+template <unsigned Tdim>
+void mpm::Mesh<Tdim>::find_particle_neighbours(const Cell<Tdim>& cell) {
```

Comment:
I intended to make the naming the same as cell, isn't it better to have similar function naming?

Reply:
I'm going to change the name in cell

Reply:
I did that